Modules/Quotas/src/Quotas.Application/DTOs/TierQuotaDefinitionDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Mapping;
+
+namespace Backbone.Modules.Quotas.Application.DTOs;
+
+public class TierQuotaDefinitionDTO : IMapTo<TierQuotaDefinition>
+{
+    public string Id { get; set; }
+    public MetricKey MetricKey { get; set; }
```

Comment:
I kicked it because I figured we wouldn't need it and, given the changes to the other domain models, the auto mapper wouldn't work with the DTO (since TierQuotaDefinition no longer has Metric but MetricKey instead).

Reply:
But just because the domain model doesn't have the reference doesn't mean that we don't need in in the DTO. That's the reason why we have DTOs. :)